great-idea/css/index-stretch.css

```
@@ -0,0 +1,468 @@
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* header {
+	position: fixed;
+	z-index: 10;
+} */
+
+header {
+  display: flex;
+}
+
+@media only screen and (max-width: 768px) {
+	header {
+		display: flex;
+		flex-direction: column;
+	}
+
+	.nav__container {
+		display: flex;
+		flex-direction: column;
+		margin: 0 auto;
+
+	}
+
+	.container {
+		display: flex;
+		flex-direction: column;
+		margin: 0 auto;
+		padding: 5px;
+		align-items: center;
+		justify-content: center;
+	}
+
+	.container li a {
+		margin: 4px;
+		padding: 3px;
+		align-items: center;
+		justify-content: center;
+		font-size: 10px;
+		width: 50px;
+		margin: 0 auto;
+		font-size: 20px;
+	}
+
+	.logo {
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		justify-content: center;
+		margin: 0 auto;
+	}
+
+	.services-header-img {
+		width: 100%;
+		height: 100%;
+	}
+
+	.services__text {
+		display: flex;
+		flex-direction: column;
+		margin: 0 auto;
+	}
+
+	.card__section {
+		display: flex;
+		flex-direction: column;
+		flex-wrap: wrap;
+		justify-content: center;
+		align-items: center;
+	}
+
+	.services--footer__container {
+		display: flex;
+		flex-flow: column wrap;
+		margin: 5px 2px;
+	}
+
+	.footer__content {
+		display: flex;
+		flex-flow: column wrap;
+		margin: 2px 2px;
+	}
+
+	.footer__content ul {
+		margin: 2px 2px;
+	}
+
+	.services-info-img {
+		width: 75%;
+		height: 75%;
+	}
+
+	.footer__content {
+		margin: 1px;
+		padding: 1px;
+	}
+
+
+	.footer__content ul {
+		margin: 2px;
+		display: flex;
+		flex-direction: column;
+		align-self: flex-start;
+		/*margin-left: -5px;*/
+		list-style: square;
+		padding: 2px;
+		margin: 0 auto;
+	}
+
+	.footer__content ul li {
+		padding: 2px;
+		margin: 2px;
+	}
+
+	.services--footer__container {
+		padding: 2px;
+		margin: 2px;
+	}
+}
+
```

for your break points to work correctly you need to have the media query with the classes something like this.
```suggestion
	header {
		display: flex;
		flex-direction: column;
		
		@media only screen and (max-width: 768px) {
		// properties here
		}
	}
```